pkg/pluginmanager/manager.go

```diff
@@ -215,6 +217,31 @@ func getPreReleasePluginDiscovery() ([]configtypes.PluginDiscovery, error) {
 		}}, nil
 }
 
+// getAdditionalTestPluginDiscoveries returns an array of plugin discoveries that
+// are meant to be used for testing new plugin version.  The comma-separated list of
+// such discoveries can be specified through the environment variable
+// "ADDITIONAL_PLUGIN_DISCOVERY_IMAGES_TEST_ONLY".
+// Each entry in the variable should be the URI of an OCI image of the DB of the
+// discovery in question.
+func getAdditionalTestPluginDiscoveries() ([]configtypes.PluginDiscovery, error) {
+	var testDiscoveries []configtypes.PluginDiscovery
+	testDiscoveryImages := strings.Split(os.Getenv(constants.ConfigVariableAdditionalDiscoveryForTesting), ",")
+	count := 0
+	for _, image := range testDiscoveryImages {
+		image = strings.TrimSpace(image)
+		if image != "" {
+			testDiscoveries = append(testDiscoveries, configtypes.PluginDiscovery{
+				OCI: &configtypes.OCIDiscovery{
+					Name:  fmt.Sprintf("test_%d", count),
```

Comment:
Any suggestion for a better name for the test discoveries?

Reply:
Do we should the name of discovery anywhere as part of our UX?

Reply:
Only in error messages.
And as a directory name in the cache:
```
$ ls ~/.cache/_tanzu/plugin_inventory/
default_pre_release test_0              test_1
```
`test_0` sounds ok in this case.